app/controllers/UploadFormController.scala

```diff
@@ -85,12 +85,7 @@ class UploadFormController @Inject()(
         case Some(uploadId) =>
              uploadProgressTracker.getUploadResult(uploadId) flatMap {
                case Some(result) if result == Quarantined => Future.successful(Redirect(routes.VirusErrorController.onPageLoad()))
-               case Some(result) =>
-                 renderer.render(
-                   "upload-result.njk",
```

Comment:
Can we remove the file upload-result.njk if it's no longer needed?

Reply:
Discussed with Butler - decided to leave it in for now as we are not 100% what is happening with this middle place holder page - Ongoing discussions with design in regards to the spin wheel etc. 

Only removing the call here for the business walk through on Thursday.